include/fuse_kernel.h

```diff
@@ -104,8 +104,8 @@
  *  - add FUSE_NO_OPEN_SUPPORT flag
  */
 
-#ifndef _LINUX_FUSE_H
```

Comment:
Well, it's deeply depressing to find out that the kernel itself violates this… I will revert that guard tonight.